projects/demo/src/modules/info/related/related.template.html

```diff
@@ -31,41 +31,60 @@
             </button>
         </a>
 
-        <a
-            href="https://github.com/taiga-family/ng-event-plugins"
-            rel="noreferrer"
-            size="l"
-            target="_blank"
-            tuiIsland
-            [hoverable]="true"
-        >
-            <h3 class="title">@tinkoff/ng-event-plugins</h3>
+        <div class="grid">
+            <a
+                href="https://taiga-family.github.io/ng-morph/"
+                rel="noreferrer"
+                size="l"
+                target="_blank"
+                tuiIsland
+                [hoverable]="true"
+            >
+                <img
+                    alt="Taiga"
```

Comment:
```suggestion
                    alt="Ng-morph logo"
```

Reply:
(+ fix other `alt`-s on this page, please)

Reply:
fix